popups/scratch-messaging/popup.css

```diff
@@ -77,27 +78,32 @@ button {
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
-  margin-right: auto;
-  margin-left: 15px;
+  margin-__MSG_@@bidi_end_edge__: auto;
+  margin-__MSG_@@bidi_start_edge__: 10px;
   max-width: 75%;
 }
 .float-right {
-  margin-left: 6px;
+  margin-__MSG_@@bidi_start_edge__: 6px;
```

Comment:
Can we use CSS properties like `margin-block-start` that are supported outside of extension contexts, except of this?

Reply:
Done. There is still one instance of `__MSG__` because `float: inline-start` and `float: inline-end` are only supported on Firefox.

Reply:
@mxmou As long as `__MSG__` is only used in non-addon code, it's OK. The main problem of using that in addon context is that TurboWarp would need to do magic to support that. And it also wouldn't work in addon userstyles with `injectAsStyleElt: true`

Reply:
@WorldLanguages It's only used in the messaging popup and the settings page. Addons handle RTL the same way as the Scratch editor itself - using `[dir="rtl"]` selectors.

Reply:
Ohh, I see. Why can't these new CSS properties be used for addons as well? Just to do it the same way as Scratch (so no need to mess with style priority)?

Reply:
> Why can't these new CSS properties be used for addons as well? Just to do it the same way as Scratch (so no need to mess with style priority)?

It's easier to work with if it's done the same way as Scratch. The main advantage of logical properties is that they use the direction of the element itself (or of its parent), but using that of `.gui` wasn't a problem in any editor addon.